File: lv0/reverse/reverse.php
<?php

function solution($my_string){
    return strrev($my_string);
}

$params = [
    ["jaron","noraj"],
    ["bread","daerb"],
];

foreach($params as $param){
    $result = solution($param[0]);
    echo "결과: {$result}, 예상결과: {$param[1]}\n";
    echo ($result == $param[1]) ? '통과' : '실패';
    echo "\n";
}

?>

File: lv0/reverse/reverse.py
def solution(my_string):
    answer = ''
    for i in my_string:
        answer = i + answer
    return answer

params = [
    ["jaron","noraj"],
    ["bread","daerb"],
]

for i in params:
    result = solution(params[0][0])
    print(result, params[0][1])
    if result == params[0][1]:
        
        print('통과')
    else:
        
        print('실패')